sphinx/builders/html.py

```diff
@@ -153,6 +153,39 @@ def __new__(cls, filename, title, rel):
         return self
 
 
+class JSContainer(list):
+    """The container for javascripts."""
```

Comment:
s/javascripts/JavaScript scripts/

Reply:
Yup, this would be more common in my experience.

Reply:
done